.yamato/ruamel/jobs/projects/commands/win.py

```diff
@@ -109,13 +109,19 @@ def install_unity_config(project_folder):
     cmds = [
         f'choco source add -n Unity -s https://artifactory.prd.it.unity3d.com/artifactory/api/nuget/unity-choco-local',
         f'choco install unity-config',
+		f'cd {TEST_PROJECTS_DIR}/{project_folder} && unity-config project set registry candidates --project-path .',
```

Comment:
we already set the registry to candidates in the main job, so we don't need this line

Reply:
(i can fix this in my branch though, so no worries)